request-cert/k8s_certificates.go

```diff
@@ -117,7 +124,7 @@ func getKubernetesCertificate(csrName string, csr []byte, wantServerAuth bool) (
 		select {
 		case event, ok := <-watchCh:
 			if !ok {
-				break
+				return nil, ChannelError
```

Comment:
Looks like we should return the same named error at the bottom of this function on line 165.

Reply:
your right, I'll make that change now